Tasks/Gradle/CodeAnalysis/SonarQube/parameter-helper.ts

```diff
@@ -0,0 +1,93 @@
+/// <reference path="../../../../definitions/vsts-task-lib.d.ts" />
+
+import tl = require('vsts-task-lib/task');
+
+import {ToolRunner} from 'vsts-task-lib/toolrunner';
+
+import {SonarQubeEndpoint} from './endpoint';
+import sqCommon = require('./common');
+
+export class SonarQubeParameterHelper {
```

Comment:
Which? C#? In Javascript there are many naming conventions. I am following jQuery's, though of course there are camelCased conventions in addition to PascalCased as you point out.